### ISSUE TITLE
Test items endpoint

### ISSUE THREAD

**Issue by jobinthomasnypl** [NYPL-discovery/discovery-general#7]
The goal of this task is to ensure that when we hit the sierra items endpoint continuously, to see if it causes any impact on the application server and encore database.

**Comment by jobinthomasnypl** [comment]
Initially this test will be run starting today at 11am with 3 threads for the next 3 days. If this looks good, then we will increase the number of threads. I will post my results here.

**Comment by jobinthomasnypl** [comment]
Test was done in 2 parts:
- Running each test's consumer with 3 threads and then with 10 threads.

Each test has two services:
1. A publisher service to be able to get the item ids that got newly created and updated and  then publish them to a queue.
2. Consumer service to get the id from the queue and query sierra api for the full item record that includes default, fixedFields and varFields and write to S3 bucket and then to solr.

Results of the test:
- No differences in resource consumption at sierra end when running using 3 thread or 10 thread for the consumer.

- However, with the publisher what was initially observed was that, when we start the publisher after few days, it was causing cpu and memory spikes (based on results provided by Antonio). This was observed couple of times based on Antonio's report. 

However, yesterday when I ran the same test (October 12th, 1:30pm), Encore (nypl-mt2.iii.com-Encore) server's impact in new relic showed cpu spike after sometime I started running the test and which shot up to more than 90% . The load also went from 15 to more than 40.  nypl-db1 server didn't have any issues and also NYPL Sierra Dataservice didn't have any issues either. So, I stopped the service within 30 - 45 mins. It was weird to see that the Encore machine show something like that while hitting the api. 

In order to confirm this, I ran the test again at 3:30pm. Surprisingly I didn't see any impact on the servers - nypl-mt2.iii.com-Encore and nypl-db1 or service - NYPL DataService.

We will have to run more tests in order to ensure that the publisher service is not causing any resource issues at sierra's end.
